walker_lake/walker_lake.ipynb

```diff
@@ -187,10 +197,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "landsat_8_img.data[landsat_8_img.data==-9999] = np.NaN  # Replace the -9999\n",
-    "ndvi8_array = (landsat_8_img[4]-landsat_8_img[3])/(landsat_8_img[4]+landsat_8_img[3])\n",
-    "ndvi8 = ndvi8_array.to_dataset(name='ndvi')[['x','y', 'ndvi']]\n",
-    "client.persist(ndvi8)"
+    "ndvi8 = (landsat_8_img[4] - landsat_8_img[3])/(landsat_8_img[4] + landsat_8_img[3])\n",
+    "# ndvi8 = ndvi8_array.to_dataset(name='ndvi')[['x','y', 'ndvi']]\n",
```

Comment:
Stray comments?

Reply:
Removed in https://github.com/holoviz-topics/examples/pull/460/commits/d2fda252708eb46df3a31c315d805a5d92d5fbbd